src/subscript/sector2fluxnum/completions.py

```diff
@@ -0,0 +1,44 @@
+from ecl2df import compdat, EclFiles
+
+
+def generate_compdat_dataframe(ECL_DATA_file_name):
```

Comment:
Is writing this function in order to simplify two Python statements down to one warranted?

Reply:
> Consider converting the wiki documentation into RST format and put it under `docs/scripts/` in this repo.

I agree, but the background disucssion of sector modelling is probably best on wiki? Do I write the rst file and use sphinx to include it in the index.html? I have not so much experience with sphinx

Reply:
Where to put docs is up to you, but put at least a starting page in RST that instructs the user to look further in the wiki.